.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = tab
+insert_final_newline = true
+trim_trailing_whitespace = false
+
+[{*.js,*.ts,}]
+indent_size = 4
+
```

Comment:
```suggestion
```

See above, this should not be needed as the indentation style is `tab` for these extensions anyway, and I want to allow readers to use their preferred tab width.